iocore/net/quic/QUICStreamManager.cc

```diff
@@ -30,7 +30,7 @@ static constexpr char tag[]                     = "quic_stream_manager";
 static constexpr QUICStreamId QUIC_STREAM_TYPES = 4;
 
 ClassAllocator<QUICStreamManager> quicStreamManagerAllocator("quicStreamManagerAllocator");
-ClassAllocator<QUICStream> quicStreamAllocator("quicStreamAllocator");
+ClassAllocator<QUICBidirectionalStream> quicStreamAllocator("quicStreamAllocator");
```

Comment:
We should probably rename this to quicBidirectionalStreamAllocator.

Reply:
Will do.